contracts/forger_stake_v2/docs/JS.md

```diff
@@ -0,0 +1,88 @@
+- `STAKE_TOTAL_VRF_PUBLIC_KEY` - **OPTIONAL**. The "VRF Public Key" from the forger to get the stake from.
+- `STAKE_TOTAL_DELEGATOR_ADDRESS` - **OPTIONAL**. Address to get stake from. If this value is provided the **STAKE_TOTAL_BLOCK_SIGN_PUBLIC_KEY** and **STAKE_TOTAL_VRF_PUBLIC_KEY** must be provided as well.
+- `STAKE_TOTAL_CONSENSUS_EPOCH_START` - **OPTIONAL**. Initial consensus epoch to calculate the stake for. Default is null.
+- `STAKE_TOTAL_MAX_NUM_OF_EPOCH` - **OPTIONAL**. Number of epochs to calculate the stake for. Default is null. If both this value and previous one are null the current consensus epoch data will be returned.
+
+### RewardsReceived operation
+In order to run the _rewardsReceived_ operation you will need to set the following environment variables:
+- `REWARDS_RECEIVED_BLOCK_SIGN_PUBLIC_KEY` - **REQUIRED**. The "Block Sign Public Key" from the forger to get the rewards from.
+- `REWARDS_RECEIVED_VRF_PUBLIC_KEY` - **REQUIRED**. The "VRF Public Key" from the forger to get the rewards from.
+- `REWARDS_RECEIVED_CONSENSUS_EPOCH_START` - **REQUIRED**. Initial consensus epoch to calculate the rewards for. 
+- `REWARDS_RECEIVED_MAX_NUM_OF_EPOCH` - **REQUIRED**. Number of epochs to calculate the rewards for.
+
+### GetForger operation
+In order to run the _getForger_ operation you will need to set the following environment variables:
+- `GET_FORGER_BLOCK_SIGN_PUBLIC_KEY` - **REQUIRED**. The "Block Sign Public Key" from the forger to get the information from.
+- `GET_FORGER_VRF_PUBLIC_KEY` - **REQUIRED**. The "VRF Public Key" from the forger to get the information from.
+
+### GetPagedForgers operation
+In order to run the _getPagedForgers_ operation you will need to set the following environment variables:
+- `GPF_START_INDEX` - **OPTIONAL**. Index to start from. Default is 0.
+- `GPF_PAGE_SIZE` - **OPTIONAL**. Page size to return. Default is 10.
+
+### GetPagedForgersStakesByDelegator operation
+In order to run the _getPagedForgersStakesByDelegator_ operation you will need to set the following environment variables:
+- `GPFS_BY_DELEGATOR_ADDRESS` - **REQUIRED**. Address to get stakes from.
+- `GPFS_BY_DELEGATOR_START_INDEX` - **OPTIONAL**. Index to start from. Default is 0.
+- `GPFS_BY_DELEGATOR_PAGE_SIZE` - **OPTIONAL**. Page size to return. Default is 10.
+
+### GetPagedForgersStakesByForger operation
+In order to run the _getPagedForgersStakesByForger_ operation you will need to set the following environment variables:
+- `GPFS_BY_FORGER_BLOCK_SIGN_PUBLIC_KEY` - **REQUIRED**. The "Block Sign Public Key" from the forger to get the stakes from.
+- `GPFS_BY_FORGER_VRF_PUBLIC_KEY` - **REQUIRED**. The "VRF Public Key" from the forger to get the stakes from.
+- `GPFS_BY_FORGER_START_INDEX` - **OPTIONAL**. Index to start from. Default is 0.
+- `GPFS_BY_FORGER_PAGE_SIZE` - **OPTIONAL**. Page size to return. Default is 10.
+
```

Comment:
It will be released a new function for the forger stake v2 contract, that should have this signature:
```
/*
   Returns the  first consensus epoch when a stake is present for a specific delegator.
   signPubKey, vrf1, vrf2 and delegator parameters are mandatory.
   If no stake has been found (the delegator never staked anything to this forger) the method returns -1
*/
function stakeStart(bytes32 signPubKey, bytes32 vrf1, bytes1 vrf2, address delegator) external view returns (int32 memory consensusEpochStart);
```
so probably it will need to be added too.